jenkins/databricks/setup.sh

```diff
@@ -54,8 +56,6 @@ fi
 # Set the path of python site-packages, and install packages here.
 PYTHON_SITE_PACKAGES="$HOME/.local/lib/python${PYTHON_VERSION}/site-packages"
 # Use "python -m pip install" to make sure pip matches with python.
-$PYSPARK_PYTHON -m pip install --target $PYTHON_SITE_PACKAGES pytest sre_yield requests pandas pyarrow findspark pytest-xdist pytest-order
+$PYSPARK_PYTHON -m pip install --target $PYTHON_SITE_PACKAGES -r ${SCRIPTPATH}/../../integration_tests/requirements.txt
```

Comment:
```
[2025-05-02T18:36:06.674Z] ++ /usr/bin/python -m pip install --target /home/ubuntu/.local/lib/python3.10/site-packages -r /home/ubuntu/../../integration_tests/requirements.txt

[2025-05-02T18:36:07.600Z] ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/home/ubuntu/../../integration_tests/requirements.txt'
```

Reply:
@razajafri saw another nightly failure and would be great to get your PR merged. The error @pxLi pointed at above is what is holding it back as far as I know.